Remotion/Web/Development.WebTesting.WebFormsControlObjects/TreeViewNodeControlObject.cs

```diff
@@ -138,8 +138,15 @@ public string GetText ()
     /// </summary>
     public bool IsChecked ()
     {
-      var checkedAttr = GetCheckboxScope()["checked"];
-      return checkedAttr != null && checkedAttr == "true";
+      try
+      {
+        var checkedAttr = GetCheckboxScope()["checked"];
+        return checkedAttr != null && checkedAttr == "true";
+      }
+      catch (MissingHtmlException exception)
+      {
+        throw new WebTestException ("The checkbox could not be found.", exception);

```

Comment:
As already noted above, this might still be a `MissingHtmlException`.

Reply:
Or a WebTestException if the TreeViewNode could be rendered without Checkbox. Not sure if thats the case, we have to check that

Reply:
https://github.com/re-motion/Trunk-SVN2Git/pull/8#discussion_r316130711